uint32 door_id
bool is_open
geometry_msgs/PoseStamped pose
